lib/network.go

```diff
@@ -1914,6 +1915,15 @@ type MsgDeSoHeader struct {
 	// event that ASICs become powerful enough to have birthday problems in the future.
 	ExtraNonce uint64
 
+	// TransactionsConnectStatus is only used for Proof of Stake blocks, starting with
+	// MsgDeSoHeader version 2. For all earlier versions, this field will default to nil.
+	//
+	// This bitset field stores information whether each transaction in the block passes
+	// or fails to connect. The bit at i-th position is set to 1 if the i-th transaction
+	// in the block passes connect, and 0 otherwise. The length of the bitset must match
+	// the number of transactions in the block.
+	TransactionsConnectStatus *bitset.Bitset
```

Comment:
Also consider naming this something like `TxnConnectStatusByIndex`, `TxnConnectStatusList`,... so the idea that the status lookup by index is contextualized in the name.